docs/leios-design/README.md

```diff
@@ -546,102 +553,89 @@ instance DSIGNAlgorithm BLS_DSIGN where
+
+## On-disk storage of ledger state
+
 > [!WARNING]
-> TODO: Identify and explain dependencies, synergies and potential conflicts with other existing or future features of the node. Potential topics:
->
-> - On-disk storage of ledger state (UTxO-/Ledger-HD)
-> - Interactions with Genesis (catching up node)
-> - Synergies with Peras:
->   - immediate: sharing key material, same key generation, registration, rotation
->   - to research: protocol-level interactions between certified EBs and boosting blocks
-> - Impact onto Mithril
-> - Which era to target and hard-fork schedule
+> TODO: Add some links and references to UTxO-HD and Ledger-HD specification and status
 
-> [!CAUTION]
-> FIXME: The remainder of this chapter is AI generated based on rough notes
+The transition from memory-based to disk-based ledger state storage represents a fundamental prerequisite for Leios deployment. This dependency stems directly from the throughput characteristics that Leios is designed to enable.
+
+At the time of writing, the latest released `cardano-node` implementation supports UTxO state storage on disk through UTxO-HD, while other parts of the ledger state including reward accounts are put on disk within the Ledger-HD initiative. The completion of this transition is essential for Leios viability, as the increased transaction volume would otherwise quickly exhaust available memory resources on realistic hardware configurations.
+
+The shift to disk-based storage fundamentally alters the resource profile of node operation. Memory requirements become more predictable and bounded, while disk I/O bandwidth and storage capacity emerge as primary constraints. Most significantly, ledger state access latency necessarily increases relative to memory-based operations, and this latency must be accounted for in the timing constraints that govern transaction validation.
+
+Early validation through comprehensive benchmarking becomes crucial to identify required optimizations in ledger state access patterns. The (re-)validation of orders of magnitude bigger transaction closures, potentially initiated by multiple concurrent threads, places particular stress on the storage subsystem, as multiple validation threads may contend for access to the same underlying state. The timing requirements for vote production - where nodes must complete endorser block validation within the $L_\text{vote}$ period - on one hand, and applying (without validation) thousands transactions during block diffusion on the other hand, impose hard constraints on acceptable access latencies.
+
+These performance characteristics must be validated empirically rather than estimated theoretically. The ledger prototyping described in the implementation plan must therefore include realistic disk-based storage configurations that mirror the expected deployment environment.
 
 ## Synergies with Peras
```

Comment:
@amesgen Is this a fair and correct summary of what we both believe the interactions and synergies to be?

Reply:
Yes, this looks good, left two minor comments.